packages/jsii-rosetta/test/translations.test.ts

```diff
@@ -87,7 +93,7 @@ for (const typeScriptTest of typeScriptTests) {
         const expected = fs.readFileSync(languageFile, { encoding: 'utf-8' });
         try {
           const translation = translator.renderUsing(visitorFactory.createVisitor());
-          expect(stripEmptyLines(translation)).toEqual(stripEmptyLines(stripCommonWhitespace(expected)));
+          expect(stripEmptyLines(translation)).toBe(stripEmptyLines(stripCommonWhitespace(expected)));
```

Comment:
?

Reply:
It's a slightly faster comparison (`===` for `toBe` versus `==` for `toEquals`), semantically equivalent for strings :)